sites/public/src/patterns/CollapsibleSection.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react"
+import PlusIcon from "@heroicons/react/24/solid/PlusIcon"
+import MinusIcon from "@heroicons/react/24/solid/MinusIcon"
+import { Heading, Icon } from "@bloom-housing/ui-seeds"
+import styles from "./CollapsibleSection.module.scss"
+
+interface CollapsibleSectionProps {
+  title: string
+  priority?: 1 | 4 | 2 | 3 | 5 | 6
```

Comment:
ope, it's used in a couple of files. Hmm, maybe it would need to be in a separate type specific file

Reply:
(Same as above)